paywall-app/vite.config.js

```diff
@@ -0,0 +1,13 @@
+// vite is only used to compile the `lib/cdn.js` file
+import { resolve } from 'path'
+import { defineConfig } from 'vite'
+
+export default defineConfig({
+  build: {
+    lib: {
+      entry: resolve(__dirname, 'lib/cdn.js'),
+      fileName: 'unlock.latest.min',
+      formats: ['cjs'],
```

Comment:
Should be umd.

I think esm should be the way to go but I suspect that will break